### ISSUE TITLE
remove analysis_interpro_parsego prop

### ISSUE THREAD

**Issue by bradfordcondon** [tripal/tripal_analysis_interpro#13]
```
  tripal_insert_cvterm(array(
    'name' => 'analysis_interpro_parsego',
    'definition' => 'Indicates whether or not the GO terms should be extracted when parsing the InterProScan XML file',
    'cv_name' => 'tripal',
    'db_name' => 'tripal',
  ));

```
this CVterm and associated property should no longer be necessary at all in tripal 3.

It also prevents GO terms for being loaded when using the v3 importer, but i would advocate for entirely removing it.

```php
      // get the DB id for the GO database
              $parsego = chado_get_property(
                  array('table' => 'analysis', 'id' => $analysis_id),
                  array('type_name' => 'analysis_interpro_parsego', 'cv_name' => 'tripal')
                  );
```

the importer pases in a `$parsego` variable form the checkbox, but overwrites it here.  probably mae sense in the context of nodes.

**Comment by bradfordcondon** [comment]
pinging this issue.  Is there an argument for keeping this?  It causes problems since my analyses dont have this property set, and its not like this module defines a custom analysis with the property there.

I still think removing it makes the most sense.  The importer has a checkbox if you should parse GO or not.  The analysisprop should have no say in the matter.

**Comment by bradfordcondon** [comment]
this was fixed in #16
